proto/proto/kvrpcpb.proto

```diff
@@ -448,77 +406,25 @@ message RawDeleteResponse {
     string error = 2;
 }
 
-message RawBatchDeleteRequest {
-    Context context = 1;
-    repeated bytes keys = 2;
-    string cf = 3;
-}
-
-message RawBatchDeleteResponse {
-    errorpb.Error region_error = 1;
-    string error = 2;
-}
-
-message DeleteRangeRequest {
-    Context context = 1;
-    bytes start_key = 2;
-    bytes end_key = 3;
-    bool notify_only = 4;
-}
-
-message DeleteRangeResponse {
-    errorpb.Error region_error = 1;
-    string error = 2;
-}
-
-message RawDeleteRangeRequest {
-    Context context = 1;
-    bytes start_key = 2;
-    bytes end_key = 3;
-    string cf = 4;
-}
-
-message RawDeleteRangeResponse {
-    errorpb.Error region_error = 1;
-    string error = 2;
-}
-
 message RawScanRequest {
     Context context = 1;
     bytes start_key = 2;
     uint32 limit = 3;
-    bool key_only = 4;
-    string cf = 5;
-    bool reverse = 6;
-    // For compatibility, when scanning forward, the range to scan is [start_key, end_key), where start_key < end_key;
-    // and when scanning backward, it scans [end_key, start_key) in descending order, where end_key < start_key.
-    bytes end_key = 7;
+    string cf = 4;
```

Comment:
If we want to be compatible with TiKV then we should not change the ids for fields, i.e., `string cf = 5`, same in the response

Reply:
no need to consider compatibility issue